utilities/infra.py

```diff
@@ -3,10 +3,12 @@
 from ocm_python_wrapper.ocm_client import OCMPythonClient
 from ocp_utilities.exceptions import (
     NodeNotReadyError,
+    NodesNotHealthyConditionError,
     NodeUnschedulableError,
     PodsFailedOrPendingError,
 )
 from ocp_utilities.infra import (
+    assert_nodes_in_healthy_condition,
     assert_nodes_ready,
```

Comment:
`assert_nodes_ready` can be removed; covered in `assert_nodes_in_healthy_condition`